### ISSUE TITLE
Possible NTR: brain lymphatic endothelial cells (also known as mural lymphatic endothelial cells or fluorescent granular epithelial cells)

### ISSUE THREAD

**Issue by LeylaR** [ZFIN/zebrafish-anatomical-ontology#177]
Should "brain lymphatic endothelial cell" be added to the AO? How do they relate to the "lymphatic endothelial cluster" already in the AO?

From Huisman et al. (PMID 34487573, ZDB-PUB-210907-11)(Meningeal lymphatic endothelial cells fulfill scavenger endothelial cell function and cooperate with microglia in waste removal from the brain. Glia. 70(1):35-49.):

"Recently, we and others have demonstrated the presence of lymphatic endothelial cells in the meningeal layer of the zebrafish brain (Bower et al., 2017a; van Lessen et al., 2017; Venero Galanternik et al., 2017). Due to the simultaneous but independent discovery of these cells, they were termed either brain lymphatic endothelial cells (BLECs) (van Lessen et al., 2017), mural lymphatic endothelial cells (Bower et al., 2017a), or fluorescent granular epithelial cells (FGP cells) (Venero Galanternik et al., 2017), with all terms referring to the identical cell type."

The Abstract:
"Brain lymphatic endothelial cells (BLECs) constitute a group of loosely connected endothelial cells that reside within the meningeal layer of the zebrafish brain without forming a vascular tubular system. BLECs have been shown to readily endocytose
extracellular cargo molecules from the brain parenchyma, however, their functional relevance in relation to microglia remains enigmatic. We here compare their functional uptake efficiency for several macromolecules and bacterial components with
microglia in a qualitative and quantitative manner in 5-day-old zebrafish embryos. We find BLECs to be significantly more effective in the uptake of proteins, polysaccharides and virus particles as compared to microglia, while larger particles like bacteria are only ingested by microglia but not by BLECs, implying a clear distribution of tasks between the two cell types in the brain area. In addition, we compare BLECs to the recently discovered scavenger endothelial cells (SECs) of the cardinal vein and
find them to accept an identical set of substrate molecules. Our data identifies BLECs as the first brain-associated SEC population in vertebrates, and demonstrates that BLECs cooperate with microglia to remove particle waste from the brain."

**Comment by LeylaR** [comment]
Abstracts of the 3 publications referenced by Huisman et al.:

Venero Galanternik et al. 2017 A novel perivascular cell population in the zebrafish brain. eLIFE. 6. 
PMID 28395729
https://zfin.org/ZDB-PUB-170412-9
"The blood-brain barrier is essential for the proper homeostasis and function of the CNS, but its mechanism of function is poorly understood. Perivascular cells surrounding brain blood vessels are thought to be important for blood-brain barrier establishment, but their roles are not well defined. Here, we describe a novel perivascular cell population closely associated with blood vessels on the zebrafish brain. Based on similarities in their morphology, location, and scavenger behavior, these cells appear to be the zebrafish equivalent of cells variably characterized as Fluorescent Granular Perithelial cells (FGPs), perivascular macrophages, or 'Mato Cells' in mammals. Despite their macrophage-like morphology and perivascular location, zebrafish FGPs appear molecularly most similar to lymphatic endothelium, and our imaging studies suggest that these cells emerge by transdifferentiation from endothelium of the optic choroidal vascular plexus. Our findings provide the first report of a perivascular cell population in the brain derived from vascular endothelium. 

Bower et al. 2017 Mural lymphatic endothelial cells regulate meningeal angiogenesis in the zebrafish. Nature Neuroscience. 20(6):774-783. 
PMID 28459441 
https://zfin.org/ZDB-PUB-170502-5
"Mural cells of the vertebrate brain maintain vascular integrity and function, play roles in stroke and are involved in maintenance of neural stem cells. However, the origins, diversity and roles of mural cells remain to be fully understood. Using transgenic zebrafish, we identified a population of isolated mural lymphatic endothelial cells surrounding meningeal blood vessels. These meningeal mural lymphatic endothelial cells (muLECs) express lymphatic endothelial cell markers and form by sprouting from blood vessels. In larvae, muLECs develop from a lymphatic endothelial loop in the midbrain into a dispersed, nonlumenized mural lineage. muLEC development requires normal signaling through the Vegfc-Vegfd-Ccbe1-Vegfr3 pathway. Mature muLECs produce vascular growth factors and accumulate low-density lipoproteins from the bloodstream. We find that muLECs are essential for normal meningeal vascularization. Together, these data identify an unexpected lymphatic lineage and developmental mechanism necessary for establishing normal meningeal blood vasculature."

van Lessen et al. 2017 Intracellular uptake of macromolecules by brain lymphatic endothelial cells during zebrafish embryonic development.  eLIFE. 6. 
PMID 28498105
https://zfin.org/ZDB-PUB-170513-1
"The lymphatic system controls fluid homeostasis and the clearance of macromolecules from interstitial compartments. In mammals brain lymphatics were only recently discovered, with significant implications for physiology and disease. We examined zebrafish for the presence of brain lymphatics and found loosely connected endothelial cells with lymphatic molecular signature covering parts of the brain without forming endothelial tubular structures. These brain lymphatic endothelial cells (BLECs) derive from venous endothelium, are distinct from macrophages, and are sensitive to loss of Vegfc. BLECs endocytose macromolecules in a selective manner, which can be blocked by injection of mannose receptor ligands. This first report on brain lymphatic endothelial cells in a vertebrate embryo identifies cells with unique features, including the uptake of macromolecules at a single cell level. Future studies will address whether this represents an uptake mechanism that is conserved in mammals and how these cells affect functions of the embryonic and adult brain."
